nidm/nidm-experiment/terms/nidm-experiment.owl

```diff
@@ -164,304 +170,898 @@ dct:title rdf:type owl:DatatypeProperty .
+                   
+                   rdfs:label "Acquisition Object"^^xsd:string ;
+                   
+                   rdfs:subClassOf :Object ;
+                   
+                   obo:IAO_0000115 "An acquisition object is an entity that represents a single stream of data, where multiple acquisitions in the same study take place simultaneously. For example, fMRI and physiological recordings."^^xsd:string ;
+                   
+                   obo:IAO_0000114 obo:IAO_0000428 ;
+                   
+                   obo:IAO_0000117 <http://purl.org/nicholsn/card.ttl> .
+
+
+
+###  http://purl.org/nidash/nidm/experiment#Acquisition_Parameter_Collection
+
+:Acquisition_Parameter_Collection rdf:type owl:Class ;
+                                  
+                                  rdfs:label "Acquisition Parameter Collection" ;
+                                  
+                                  rdfs:subClassOf prov:Collection .
+
+
+
+###  http://purl.org/nidash/nidm/experiment#AnatomicalImageAcquisition
+
+:AnatomicalImageAcquisition rdf:type owl:Class ;
+                            
+                            rdfs:subClassOf :MRI_Acquisition .
 
 
 
 ###  http://purl.org/nidash/nidm/experiment#AnatomicalScan
 
-nidme:AnatomicalScan rdf:type owl:Class ;
-                    
-                    rdfs:subClassOf prov:Entity .
+:AnatomicalScan rdf:type owl:Class ;
+                
+                rdfs:subClassOf :AcquisitionObject .
 
 
 
-###  http://purl.org/nidash/nidm/experiment#AnatomyImagingAcquisition
+###  http://purl.org/nidash/nidm/experiment#Attribute
 
-nidme:AnatomyImagingAcquisition rdf:type owl:Class ;
-                               
-                               rdfs:subClassOf prov:Activity .
+:Attribute rdf:type owl:Class ;
+           
+           rdfs:subClassOf prov:Entity .
 
 
 
 ###  http://purl.org/nidash/nidm/experiment#BaselineProtocol
 
-nidme:BaselineProtocol rdf:type owl:Class ;
-                      
-                      rdfs:label "Baseline Protocol"^^xsd:string ;
-                      
-                      rdfs:subClassOf prov:Plan ;
+:BaselineProtocol rdf:type owl:Class ;
+                  
+                  rdfs:label "Baseline Protocol"^^xsd:string ;
+                  
+                  rdfs:subClassOf prov:Plan ;
+                  
+                  obo:IAO_0000115 "A baseline protocol is a plan that represents the study design for a given project."^^xsd:string ;
+                  
+                  obo:IAO_0000114 obo:IAO_0000428 ;
+                  
+                  obo:IAO_0000117 <http://purl.org/nicholsn/card.ttl> .
+
+
+
+###  http://purl.org/nidash/nidm/experiment#BehaviorAndConditionOnsets
+
+:BehaviorAndConditionOnsets rdf:type owl:Class ;
+                            
+                            rdfs:subClassOf prov:Entity .
+
+
+
+###  http://purl.org/nidash/nidm/experiment#Behavioral_File_Format
+
+:Behavioral_File_Format rdf:type owl:Class ;
```

Comment:
For new terms, the syntax we've been using for classes is `BehavioralFileFormat` without underscores - we only use underscores when importing external terms that already use this convention